core/client/app/components/gh-upload-modal.js

```diff
@@ -22,6 +24,19 @@ export default ModalDialog.extend({
             this.$('.js-upload-url').removeClass('error');
         }
     },
+    addKeyboardShortcutHandler: function () {
+        var self = this;
+
+        this.$(document).on('keydown', function (event) {
+            event.preventDefault();
+            if (event.keyCode === keyboardCodes.enter) {
+                self.get('controller').send('confirm', 'accept');
+            }
+        });
+    },
+    willDestroyElement: function () {
+        this.$(document).off('keydown');
```

Comment:
Hi @kevinansfield. Thanks for the feedback! This is a pretty naive attempt, attaching the `keydown` handler to `document`, hence why it needs to remove all listeners. I'll take your comments on board for another iteration. :smile: